app/src/test/java/com/x8bit/bitwarden/data/auth/datasource/disk/AuthDiskSourceTest.kt

```diff
@@ -295,6 +295,10 @@ class AuthDiskSourceTest {
         )
         authDiskSource.storeEncryptedPin(userId = userId, encryptedPin = "encryptedPin")
         authDiskSource.storeMasterPasswordHash(userId = userId, passwordHash = "passwordHash")
+        authDiskSource.storeAuthenticatorSyncUnlockKey(
+            userId = userId,
+            authenticatorSyncUnlockKey = "key",
```

Comment:
Minor, but following the pattern of this test would suggest this value should be `"authenticatorSyncUnlockKey"`